src/Form/ColorChoiceType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ColorChoiceType extends AbstractType
+{
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefaults([
+            'choice_label' => function ($value) {
```

Comment:
This form type could be more flexible if the parent decides of this behavior. It could be achieved with `'choices' => array_combine($values, $values)`.

Reply:
hm, it's juste default value for the option, it can be easely overridden by `add('name', ColorChoiceType::class, ['choice_label' => function() {...}])`

Reply:
You are right 👍